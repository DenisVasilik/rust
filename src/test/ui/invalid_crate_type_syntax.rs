// regression test for issue 16974
#![crate_type(lib)]  //~ ERROR attribute must be of the form `#[crate_type = "..."]`

fn my_lib_fn() {}

fn main() {}
