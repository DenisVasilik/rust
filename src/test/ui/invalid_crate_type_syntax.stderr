error: attribute must be of the form `#[crate_type = "..."]`
  --> $DIR/invalid_crate_type_syntax.rs:2:1
   |
LL | #![crate_type(lib)]  //~ ERROR attribute must be of the form `#[crate_type = "..."]`
   | ^^^^^^^^^^^^^^^^^^^

error: aborting due to previous error

